### DISCUSSION TITLE
How to display images as Base64 format

### DISCUSSION THREAD

**Discussion by condorheroblog** [hunghg255/reactjs-tiptap-editor#41] [Ideas]
Whether exporting PDF or Word files, the image is a link(like this https://picsum.photos/1920/1080.webp?t=1) and cannot be used directly. We need to make a second request.

Is there a way that when the rich text editor displays the content, the developer manually sends the request image as Base64 format, but the API for calling the rich text editor is still a URL, so that there is no need to make a second request when exporting the file, and it does not affect the display of the text and image content.

Do you have any good ideas?

**Comment by hunghg255** [comment]
@condorheroblog I do not have an idea, currently I think the best practice we should handle from the backend

**Comment by condorheroblog** [comment]
See this:

https://codesandbox.io/p/sandbox/get-url-show-base64-dcwclp
